java/code/src/com/redhat/rhn/frontend/action/common/DownloadFile.java

```diff
@@ -362,7 +362,7 @@ private ActionForward handleUserDownload(HttpServletRequest request, String url,
     protected StreamInfo getStreamInfo(ActionMapping mapping, ActionForm form,
             HttpServletRequest request, HttpServletResponse response) throws Exception {
 
-        String path;
+        String path = "";
```

Comment:
Rather than setting this variable to `""` remove it from here and declare it where it is used.